src/blog/drizzle-150-a-new-beginning.md

```diff
@@ -0,0 +1,63 @@
+
+## From many to one
+
+Today, we are announcing that we are moving to a monorepo. From now on, a single PR can be used to update code and documentation that used to span multiple repos.
+
+Over the last two years, we’ve developed libraries on top of Drizzle such as `drizzle-react` and `drizzle-react-components`. Since these libraries depend so heavily on each other, it only makes sense for us to move all of these packages to a monorepo to streamline any updates.
+
+In addition, we now also have an experimental Vue.js plugin that fits naturally into the monorepo architecture. It is very likely that going forward we will develop new packages to aid the Drizzle ecosystem in this manner.
+
+## Scoped package names
+
+Another big change we are making is towards scoped package names on NPM. This means that instead of:
+
+```
+drizzle
+drizzle-react
+drizzle-react-components
+drizzle-vue-plugin
+```
+
+We will now have:
+
+```
+@drizzle/store
+@drizzle/react-plugin
+@drizzle/react-components
+@drizzle/vue-plugin
+```
+
+From an organizational standpoint, this makes it a lot more intuitive to manage your dapp development.
+
+## Testing
+
+Prior to the monorepo, testing our libraries together was a slow, error-prone and painful experience. Our maintainers had to build custom dapps to isolate changes and manually link packages together as well as coordinate with ganache for each front end.
+
+The monorepo allows us to have dedicated front end apps that automatically build against the latest libraries and we can conveniently orchestrate them with a single ganache instance in just a couple commands. In other words, you can easily spin up three apps to test various aspects of the monorepo [in seconds](https://github.com/trufflesuite/drizzle/blob/develop/README.md#testing-and-debugging-with-ganache).
+
+<figure>
+  <img src="/img/blog/drizzle-150-a-new-beginning/test-ui.png" alt="Test UI instances" style="width:100%">
+  <figcaption style="text-align: center;">_Easily spin up real front ends to test that dapps can work with the current code_</figcaption>
+</figure>
+
+## Better release strategy
+
+Perhaps the biggest thing that the changes above allow us to do is more frequent releases.
+
+In the past year, we often had to deal with delaying the publishing of a new change because it required testing and updating other repos. Our publishing process was essentially being blocked by having to synchronize changes amongst multiple repos.
+
+With a monorepo that all changes, it means that you can look forward to more regular releases and a faster turnaround time for getting issues fixed. With each release, better changelogs are also coming, as we move towards adopting [conventional commits](https://www.conventionalcommits.org) for each PR that gets merged.
```

Comment:
This sentence is really hard to parse. 

> With a monorepo that all changes, it means that you can look forward to more regular releases and a faster turnaround time for getting issues fixed.

I got stopped up after the word "monorepo", and needed a comma there to better understand the sentence. Also, the use of "it means" after the current comma doesn't flow well. I'd change to one of the following options (my changes bolded): 

1. With a monorepo, that all changes, **which** means that you can look forward to more regular releases and a faster turnaround time for getting issues fixed.
1. With a monorepo, that all changes, **which** means ~that you can look forward to~ more regular releases and a faster turnaround time for getting issues fixed.
1. With a monorepo, that all changes, **meaning** ~that you can look forward to~ more regular releases and a faster turnaround time for getting issues fixed

Something like that. :) 

Also, this line can be stronger: 

> With each release, better changelogs are also coming, as we move towards adopting [conventional commits](https://www.conventionalcommits.org) for each PR that gets merged.

I'd change it to: 

1. Better changelogs are also coming as we move toward adopting conventional commits for each PR that gets merged.

Also note your `towards` should be `toward` (no 's') 

Sorry this looks pretty anal. Great post! 😄

Reply:
Thanks! Updated.